universal-login-relayer/test/core/services/execution/PendingMessages.test.ts

```diff
@@ -110,15 +110,15 @@ describe('INT: PendingMessages', () => {
       await pendingMessages.add(signedMessage);
       await messageRepository.markAsPending(messageHash, '0x829751e6e6b484a2128924ce59c2ff518acf07fd345831f0328d117dfac30cec', '2020');
       const status = await pendingMessages.getStatus(messageHash);
-      const required = await (await walletContractService.getRequiredSignatures(signedMessage.from)).toNumber();
+      const required = (await walletContractService.getRequiredSignatures(signedMessage.from)).toNumber();
```

Comment:
why duble await?

Reply:
I don't know. I reduced it

Reply:
Aaaa sorry, I thought that you added the second await 😅